src/address.js

```diff
@@ -64,9 +65,17 @@ export function isCurrentAddressType(address, addressType) {
 
 /** PUBLIC **/
 
-export function isAddressComplete(address) {
+export function isAddressComplete(address, options) {
   const { number, street, neighborhood, city, state } = address || {}
-  return !!(number && street && neighborhood && city && state)
+  const verifyNumber = !(get(options, 'verifyNumber') === false)
```

Comment:
If `verifyNumber` is a boolean var, so:
1 - If `get(options, 'verifyNumber')` is false, then the statement will be `!(false=== false)` => `!(true)`=> `false`
2 - if `get(options, 'verifyNumber')` is true, then => `!(true === false)`=> `!(false)` => `true `
```suggestion
  const verifyNumber = get(options, 'verifyNumber')
```

Reply:
It is not always boolean. I will create a default param.

Reply:
So you can pass a default value as the third parameter in get, something like
 `get(options, 'verifyNumber', false)`